src/components/pages/events/CheckoutSuccess.js

```diff
@@ -842,6 +845,7 @@ class CheckoutSuccess extends Component {
 						</div>
 					</Dialog>
 				</Hidden>
+				<Drift appId={appId}/>
```

Comment:
If there is no `appId` what happens?
```suggestion
				{ appId && <Drift appId={appId}/> }
```